src/pack/index.ts

```diff
@@ -12,6 +12,10 @@ import { Blockstore } from '../blockstore/index'
 import { MemoryBlockStore } from '../blockstore/memory'
 
 export async function pack ({ input, blockstore: userBlockstore }: { input: ImportCandidateStream, blockstore?: Blockstore }) {
+  if (!input || (Array.isArray(input) && !input.length)) {
+    throw new Error('given input could not be parsed correctly')
```

Comment:
This error message could be better I feel!

e.g. "missing input file(s)"

Reply:
```suggestion
    throw new Error('missing input file(s)')
```